projects/igniteui-angular/src/lib/services/excel/excel-exporter-grid.spec.ts

```diff
@@ -908,6 +911,82 @@ describe('Excel Exporter', () => {
         });
     });
 
+    describe('', () => {
+        beforeEach(waitForAsync(() => {
+            options = createExportOptions('GridMultiColumnHeadersExcelExport');
+        }));
+
+        it('should export grid with multi column headers', async () => {
+            const fix = TestBed.createComponent(MultiColumnHeadersExportComponent);
+            fix.detectChanges();
+
+            const grid = fix.componentInstance.grid;
+
+            await exportAndVerify(grid, options, actualData.exportMultiColumnHeadersData);
+        });
+
+        it('should export grid with multi column headers and moved column', async () => {
+            const fix = TestBed.createComponent(MultiColumnHeadersExportComponent);
+            fix.detectChanges();
+
+            const grid = fix.componentInstance.grid;
+
+            grid.columns[0].move(3);
+            fix.detectChanges();
+
+            await exportAndVerify(grid, options, actualData.exportMultiColumnHeadersDataWithMovedColumn);
+        });
+
+        it('should export grid with hidden column', async () => {
+            const fix = TestBed.createComponent(MultiColumnHeadersExportComponent);
+            fix.detectChanges();
+
+            const grid = fix.componentInstance.grid;
+            grid.columns[0].hidden = true;
+            fix.detectChanges();
+
+            expect(grid.visibleColumns.length).toEqual(15, 'Invalid number of visible columns!');
+            await exportAndVerify(grid, options, actualData.exportMultiColumnHeadersDataWithHiddenColumn);
+        });
+
+        it('should export grid with hidden column and ignoreColumnVisibility set to true', async () => {
+            const fix = TestBed.createComponent(MultiColumnHeadersExportComponent);
+            fix.detectChanges();
+
+            const grid = fix.componentInstance.grid;
+            grid.columns[0].hidden = true;
+            options.ignoreColumnsVisibility = true;
+            fix.detectChanges();
+
+            await exportAndVerify(grid, options, actualData.exportMultiColumnHeadersData);
+        });
+
+        it('should export grid with pinned column group', async () => {
+            const fix = TestBed.createComponent(MultiColumnHeadersExportComponent);
+            fix.detectChanges();
+            await wait();
+
+            const grid = fix.componentInstance.grid;
+            grid.columns[1].pinned = true;
+            fix.detectChanges();
+
+            expect(grid.pinnedColumns.length).toEqual(5, 'Invalid number of pinned columns!');
```

Comment:
Unnecessary verification. This should have been covered in MCH tests with pinning.